.circleci/config.yml

```diff
@@ -72,7 +72,7 @@ commands:
       - run: yarn pretest
       - run:
           name: Run node tests
-          command: yarn test.jest --coverage --maxWorkers=<< parameters.max-workers >>
```

Comment:
This removal of `--coverage` intentional? (ok if it is, just double checking)

Reply:
Yeah, it's already gone in develop branch